CombinatorialGames/Test/GameCmp.lean

```diff
@@ -27,6 +27,11 @@ example : (2 : IGame) + 2 ≈ 4 := by game_cmp
 example : (3 : IGame) - 2 ≈ 1 := by game_cmp
 example : (2 : IGame) * 2 ≈ 4 := by game_cmp
 
+-- Int/Nat decidability
+example : (200 : IGame) < 400 := by game_cmp
+example : (-200 : IGame) ≤ 400 := by game_cmp
+example : (-200 : IGame) ≈ -200 := by game_cmp
```

Comment:
These tests are also a bit misleading, they work just fine without even having to rely on decidability! (The top one does time out currently)